Framework/Algorithms/src/ReflectometryReductionOne2.cpp

```diff
@@ -425,12 +430,49 @@ ReflectometryReductionOne2::validateInputs() {
   const auto transmission = validateTransmissionProperties();
   results.insert(transmission.begin(), transmission.end());
 
+  const auto output = validateOutputWorkspaces();
+  results.insert(output.begin(), output.end());
+
+  return results;
+}
+
+/** Validate output workspaces.
+ */
+std::map<std::string, std::string>
+ReflectometryReductionOne2::validateOutputWorkspaces() const {
+  std::map<std::string, std::string> results;
+  bool const isDebug = getProperty("Debug");
+  if (!isDebug && !isChild()) {
+    if (isDefault("OutputWorkspace")) {
+      results["OutputWorkspace"] =
+          "OutputWorkspace property must be set when Debug is not enabled.";
```

Comment:
I don't think we need to enforce this because we can use the default name. Sorry if I didn't explain this well in the initial issue.